jest-resolver.js

```diff
@@ -0,0 +1,14 @@
+module.exports = (request, options) => {
```

Comment:
**Question**: I haven't seen this in the guide to migrate origami components. Is this needed on all migrations from bower to npm?

Reply:
So I had an issue when running the Jest tests with the new updated components. Specifically, the `o-overlay` component has published only the Browserify entry point (`"browser": '"main.js" [here](https://github.com/Financial-Times/origami/blob/8e60e8210c871b9adaf4f09a4b92f1a49ef4ce56/components/o-overlay/package.json#L21) and Jest can't import properly. In order to fix this found a solution by writing a Jest resolver that will include the `browser` field from all packages.

Reply:
thank you. I will keep an eye out for future repos.